clues.js

```diff
@@ -13,6 +13,34 @@ module.exports = {
       "photographer:"
     ]
   },
+  exactMatch: {
+    fn: (item, alt) => item == alt.trim(),
+    message: value => `Alt text should not be "${value}".`,
+    rules: [
+      "image",
+      "graphic",
+      "photo",
+      "photograph",
+      "placeholder",
+      "temp",
+      "alt",
+      "drawing",
+      "painting",
+      "artwork",
+      "logo",
+      "bullet",
+      "button",
+      "arrow",
+      "more",
+      "spacer",
+      "blank",
+      "chart",
+      "table",
+      "diagram",
+      "graph",
+      "*"
```

Comment:
@jsnmrs Want to add `" "` and remove the `checkOnlySpace` function and tests?

Reply:
We'll keep that test separate – the `trim()` will come into play